src/app.js~

```diff
@@ -0,0 +1,70 @@
+go.app = function() {
```

Comment:
`app.js~` is also a temporary file from your editor that should be removed from git.